Monika After Story/game/zz_calendar.rpy

```diff
@@ -2274,3 +2264,23 @@ init python:
         """
         if not mas_current_background.hide_calendar and not mas_calIsVisible_ovl():
             renpy.show_screen("calendar_overlay", _layer="master")
+
+    def mas_isLeapYear(year):
```

Comment:
actually the better way to do this is just try to build the Feb 29 date object and catch the exception. (See `add_years` in `definitions`)